### DISCUSSION TITLE
You guys have gone overboard

### DISCUSSION THREAD

**Discussion by rlovell3** [mylinuxforwork/dotfiles#436] [General]
I have been installing/testing various flavors of the ml4w dotfiles for months now.  At some point, you began symlinking basically everything that controls the shell to ~/dotfiles/config/... making user customizations unlike anything any of us have been doing for the past 25 years.  I, unfortunately, must part ways with this project.  Doing something as simple as tweaking my .zshrc file, or using a custom oh-my-zsh config file, no longer is anything like the two dozen other Linux machines in my lab. It seems everything is sym-mapped to the dotfiles, and  on some of my machines, those get updated, while on others, they don't.  Your earlier work was fantastic.  But the recent stuff is just out of control, and unfortunate.  Good luck with your project, but I have reached a point where I must check out.  Regards, RL.

**Comment by mylinuxforwork** [comment]
@rlovell3 Thank you bery much for your feedback. 
But I really don’t understand your issues. Symlinks into the dotfiles was the concept from the beginning.

I assume that some of your individual customizations have been overwritten with the update. I am currently working on 2.9.7 and are improving the installation script and will provide support for Fedora.

Your feedback with more details would be really helpful to improve the installation script and with that you can maybe give it another chance.

Thanks in advance,

Stephan
